spec/eth/address_spec.rb

```diff
@@ -125,6 +123,28 @@
     end
   end
 
+  describe ".null?" do
```

Comment:
Maybe another good name or alias would be `.zero?`. I figure `.null?` is safer since we use `nil` in ruby instead.

Reply:
Yes, I would prefer zero. :+1: